core/src/main/java/com/graphhopper/GraphHopper.java

```diff
@@ -1015,6 +1045,10 @@ public Weighting createTurnWeighting( Graph graph, FlagEncoder encoder, Weightin
         return weighting;
     }
 
+    public BBox getBounds(){
```

Comment:
I use that in the InfoServlet to return the correct BBox to the user. Right now you can get the storage, that has a getBounds method as well.